### ISSUE TITLE
Something broken?

### ISSUE THREAD

**Issue by kshpytsya** [crdoconnor/strictyaml#17]
I encounter the following error in a freshly created virtual environment for both 2.7 and 3.6 versions of Python. This happens on two different machines, Gentoo and Ubuntu.

```
>>> import strictyaml as sy
>>> sy.load("1: a")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/VENV2/lib/python2.7/site-packages/strictyaml/parser.py", line 215, in load
    document = ruamelyaml.load(yaml_string, Loader=StrictYAMLLoader)
  File "/tmp/VENV2/lib/python2.7/site-packages/ruamel/yaml/main.py", line 95, in load
    loader = Loader(stream, version, preserve_quotes=preserve_quotes)
  File "/tmp/VENV2/lib/python2.7/site-packages/strictyaml/parser.py", line 200, in __init__
    RoundTripScanner.__init__(self)
  File "/tmp/VENV2/lib/python2.7/site-packages/ruamel/yaml/scanner.py", line 91, in __init__
    self.fetch_stream_start()
  File "/tmp/VENV2/lib/python2.7/site-packages/ruamel/yaml/scanner.py", line 407, in fetch_stream_start
    mark = self.reader.get_mark()
  File "/tmp/VENV2/lib/python2.7/site-packages/ruamel/yaml/scanner.py", line 135, in reader
    return self.loader._reader
AttributeError: 'NoneType' object has no attribute '_reader'
```

**Comment by kshpytsya** [comment]
Apparently ruamel.yaml 0.14.0 causes the breakage.

**Comment by crdoconnor** [comment]
Thanks, looking into it now.

**Comment by crdoconnor** [comment]
Fixed in version 0.5.2, thanks @kshpytsya !
